Cargo.toml

```diff
@@ -34,8 +34,8 @@ triggered = "0.1.2"
 prost = "0.12"
 
 [dev-dependencies]
-bitcoincore-rpc = { package="core-rpc", version = "0.17.0" }
-bitcoind = { version = "0.30.0", features = [ "22_0" ] }
+bitcoincore-rpc = "0.19.0"
+corepc-node = { version = "0.7.0", features = [ "27_2", "download"] }
```

Comment:
Any reason we can't bump to bitcoin core 28 with the [`28_0` feature](https://github.com/rust-bitcoin/corepc/blob/2643b9d5c0bfaf3277848eba22cff6c9b8a16b96/node/Cargo.toml#L48)?

I see since April 14th, v29 is also out, but that's too recent for the feature to be available in `corerpc-node`, I guess!

Reply:
oh no, I dig a bit more on this and it is just our version of lnd. I feel like I'm in a spiral of dependencies.
In another PR we can upgrade lnd tag and move to 28_0, so we can merge this. What do you think @dunxen?

Reply:
yeah, no stress! was just curious :)